### PR TITLE
Updated link to support passing in classname. Added in components-lis…

### PR DESCRIPTION
**Pull request by roblevintennis**
https://www.pivotaltracker.com/story/show/163408640

### CODE REVIEW

File: styleguide/components/components-list/components-list.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  return (
+    <ul className={styles.list}>
+      {filteredItems.map(({ name, href, content }) => {
+        if (current === name) {
+          console.log(`current: ${current}`);
+          console.log(`content: ${content}`);
+          console.log(`href: ${href}`);
+          console.log(`name: ${name}`);
```
Comment (ahuth) [inline styleguide/components/components-list/components-list.jsx]: Hey @roblevintennis @juanca these console logs caused the build to fail on lint checks. Do we still want these in here?

Also, FYI, one thing I like doing when debugging is grouping all these logs into one thing:

```js
console.table({ current, content, href, name });
```

Makes it pretty print real nicely.

Comment (juanca) [reply styleguide/components/components-list/components-list.jsx]: Good call. I'll set up another PR to fix this. It also seems like we need to tighten the requirements for merging into `master`. 👀

Comment (ahuth) [reply styleguide/components/components-list/components-list.jsx]: Thanks. Also looks like I never set up a GitHub integration with CI 😬 

Could make it easier to spot these issues. I can see if I have access to do that

Comment (juanca) [reply styleguide/components/components-list/components-list.jsx]: I believe only admin members of the owning organization can change those settings. Done and done @ https://github.com/mavenlink/design-system/pull/13

**Pull request merged**